package/kernel/linux/modules/netdevices.mk

```diff
@@ -1042,3 +1042,47 @@ define KernelPackage/be2net/description
 endef
 
 $(eval $(call KernelPackage,be2net))
+
+define KernelPackage/mlx4
```

Comment:
might be personal preference but the name of the package should be the same as the kernel modules name

Reply:
fix